CMakeLists.txt.user

```diff
@@ -0,0 +1,1167 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Delete this file!!

Reply:
I didn't use "git add *" this time to try and avoid adding that file back in again
I'm going to write a .gitignore for all .user files tomorrow, that should solve the problem permanently without ignoring anything important right?